packages/data-context/src/sources/ProjectDataSource.ts

```diff
@@ -478,6 +479,18 @@ export class ProjectDataSource {
     return preferences[projectTitle] ?? null
   }
 
+  async getCodeGenGlobs () {
+    assert(this.ctx.currentProject, `Cannot find glob without currentProject.`)
+
+    const looseComponentGlob = '*.{js,jsx,ts,tsx,.vue}'
+
+    const detected = await detectFramework(this.ctx.currentProject || '')
```

Comment:
The `config.component.devServer` key is a more valid indicator for what the project is using vs `detectFramework`. I don't think we need to use `detectFramework` at all as part of this work.